<launch>
    <!-- <arg name="marker_size_frontleft" default="7.62" /> -->
    <!-- <arg name="marker_size_frontleft_2" default="7.62" /> -->
    <!-- <arg name="marker_size_frontright" default="7.62" /> -->
    <!-- <arg name="use_bundle" default="true" /> -->


    <!-- Launch all nodes to control Spot, simulated or real, using MoveIt -->
    <include file="$(find spot_skills)/launch/spot_moveit_bringup.launch" pass_all_args="true" />

    <!-- Run Rviz using a config file designed for this demo -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find spot_skills)/rviz/moveit_spot_demo.rviz" />

    <!-- Launch the AR marker detection -->
    <include file="$(find spot_skills)/launch/detect_markers_bundle.launch" />
    <!-- <include file="$(find spot_skills)/launch/detect_markers_single.launch" /> -->
        <!-- <arg name="marker_size_frontleft" value="$(arg marker_size_frontleft)" /> -->
        <!-- <arg name="marker_size_frontleft_2" value="$(arg marker_size_frontleft_2)" /> -->
        <!-- <arg name="marker_size_frontright" value="$(arg marker_size_frontright)" /> -->
        <!-- <arg name="use_bundle" value="$(arg use_bundle)" /> -->
    <!-- </include> -->

    <node name="object_pose_estimator" pkg="spot_skills" type="obj_pose_est.py" output="screen" />
</launch>